### ISSUE TITLE
blocksNotToPillarOn

### ISSUE THREAD

**Issue by beesyndicate** [cabaletta/baritone#876]
## Describe your suggestion
I have been streaming an AI bot building things from schematics. I have noticed a few issues that have come to light. I don't know if here is the best place to put these things but if there is a more appropriate place please let me know.  

The AI has trouble pillering on blocks that are not full blocks. (In my case GlassPaynes) It would be great if a command could be added that would be something like 

"blocksNotToPillarOn"

This would cause the AI to pick another path or pillar next to the dissalowed type with a throw away block. 

as of right now the AI will do things like place down a torch, glass payne, or iron bar underneath his feet get on top of it then break the block it just placed causing it to essentially hop in place indefinably.  

## Settings
add block types to avoid placing directly under the players feet and assend.

## Context
more reliable building from schematics 

[https://clips.twitch.tv/FunnyIgnorantBeanPermaSmug](url)

## Final checklist
- [ ] I have not used any OwO's or UwU's in this issue.

**Comment by Orinion** [comment]
https://clips.twitch.tv/DistinctOilyOcelotStrawBeary

**Comment by beesyndicate** [comment]
yes sometimes he breaks the block beneath the target block too. In creative mode as in the clip from Orinion this presents another set of problems :(

**Comment by usereewrjrjeejr** [comment]
Stop using baritone in creative

**Comment by beesyndicate** [comment]
> 
> 
> Stop using baritone in creative

Why?

**Comment by mrphax0r** [comment]
Because baritone is designed for survival mode and has no support for creative whatsoever.
